build/style.css

```diff
@@ -100,3 +104,32 @@ strong {
 	opacity: 0.6;
 	border-bottom-color: #bb5536;
 }
+
+div.main.wrapper h2 {
+	color: #b30000;
+}
+
+div.wrapper h2.trouble {
```

Comment:
rename to `h2.trouble` ... it doesn't matter whether or not it is inside a wrapper div. We want it to appear the same no matter what HTML around it changes.